src/core/resources/qml/WallContentWindow.qml

```diff
@@ -0,0 +1,46 @@
+import QtQuick 1.1
+import DisplayCluster 1.0
+
+Rectangle {
+    id: rootObj
+
+    color: "transparent"
+//    border.color: "black"
```

Comment:
Not until rendering is done in QML. I guesss I should have deleted it for now instead of commenting.